docs/presentations/articles/rasa.md

```diff
@@ -0,0 +1,132 @@
+# Rasa
+
+_Author: [Andreas Loeffler](mailto:an161loe@htwg-konstanz.de)_
+
+## TL;DR
+
+_Rasa_ is an open-source conversational AI framework with two main components: Rasa NLU for natural language understanding and Rasa Core for dialogue management. It stands out with its contextual understanding, scalability, and open-source flexibility.
+
+## What is Rasa?
+
+[Rasa](https://rasa.com/) is an open-source conversational AI framework that enables developers to build robust and natural language understanding (NLU) models for chatbots and virtual assistants. It goes beyond simple rule-based approaches and allows for the creation of intelligent, context-aware conversational agents.
+
+## Key Components of Rasa
+
+### 1. Rasa NLU
```

Comment:
imho, no need for enumeration. It looks a bit weird in the ToC to the right, if level 3 sections are enumrated, but the others not.